src/SFA.DAS.Employer.PR.Web/Controllers/RegisteredProvidersController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Employer.PR.Domain.Interfaces;
+using SFA.DAS.Employer.PR.Domain.Models;
+using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Services;
+
+namespace SFA.DAS.Employer.PR.Web.Controllers;
+
+public class RegisteredProvidersController(IOuterApiClient outerApiClient, ICacheStorageService cacheStorageService) : Controller
+{
+    [HttpGet("registeredProviders")]
```

Comment:
Auth is missing here

Reply:
auth breaks this. This is used in a javascript call, so auth stops it working.  I'll double check but I'm pretty sure whereever we use this approach in textbox population, we don't add authentication to the controller doing such work,.

Reply:
I'm not correct, elsewhere (eg admin aan) we are using auth.  However, it breaks it.  I will investigate

Reply:
I added [Authorize] is that enough? If I put in more specific roles, it seems to stop working when the javascript calls it